src/plugins/explore/public/components/query_panel/components/footer/show_input_type.tsx

```diff
@@ -41,7 +43,8 @@ export const ShowInputType: React.FC<ShowInputTypeProps> = ({
       className="queryPanel__footer__showInputType"
       data-test-subj="queryPanelFooterShowInputType"
     >
-      {getDisplayText}{' '}
+      {getDisplayText}
+      {getDisplayText.includes(LanguageType.PPL) && pplLanguageReference(LanguageType.PPL)}
```

Comment:
the function `pplLanguageReference` gets `PPL` as an argument? Can we either do:

- make `pplLanguageReference()` generic (by removing `ppl` in the name) and having a switch statement that switches based on the `selectedLanguage` argument and displays the correct language reference (recommended)
- Remove the `selectedLanguage` argument since it already assumes that it is on PPL (via the name)